### DISCUSSION TITLE
I need advice on trying to create presets for Analog Drone and Sony Handycam Camcorder

### DISCUSSION THREAD

**Discussion by Asyahril395** [ntsc-rs/ntsc-rs#136] [Q&A]
I need advice on trying to create presets for Analog Drone and Sony Handycam Camcorder, any advice?
Drone Analog like this:
![Drone Analog 2](https://github.com/user-attachments/assets/b16ad9b7-77a9-4135-ae1a-8c65f7b19824)
![Drone Analog 5](https://github.com/user-attachments/assets/06492658-4c80-47ba-9fe8-d21d279f7a6a)
![PICT0022](https://github.com/user-attachments/assets/7983bacd-f527-4069-9b15-4544771f274b)

And Sony Handycam Camcorder like this:
![Sony Handycam 4](https://github.com/user-attachments/assets/68f533a8-6011-474e-bca8-6a9fa406d8d8)
![Sony Handycam 5](https://github.com/user-attachments/assets/122ecc32-a7d3-47ef-8316-36e8df580f4a)
![Sony Handycam 1](https://github.com/user-attachments/assets/30c6240c-140c-49e5-b033-23d7f11c9b40)
![Sony Handycam 2](https://github.com/user-attachments/assets/38f29d2e-6b2a-4034-9100-090ec5e55594)

Sorry, there are quite a lot of example photos...
Anyone can help? Thanks! (ᴖ◡ᴖ)

**Comment by Degamisu** [comment]
For the handycam, A preset was created, but could use some more research!

#132 

The analog one might seem like a fun challenge. I'd recommend taking example photos and having them side by side, going down the list of each setting and finetuning it.

**Reply by Asyahril395** [reply]
Oh yeah, thank you! (⋂‿⋂)

**Comment by dawnch0rus** [comment]
>Drone Analog like this: ![Drone Analog 2](https://private-user-images.githubusercontent.com/176497054/351752663-b16ad9b7-77a9-4135-ae1a-8c65f7b19824.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4NTk0MzYsIm5iZiI6MTcyMTg1OTEzNiwicGF0aCI6Ii8xNzY0OTcwNTQvMzUxNzUyNjYzLWIxNmFkOWI3LTc3YTktNDEzNS1hZTFhLThjNjVmN2IxOTgyNC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNFQyMjEyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYjU5MTY1ZDU2ZGU3NmIwZWQ4NGRhNmU5NjJlMmFlZTRkNDI2MDMyY2MzODI5MTc3N2Q3YmM3NmQwNGZjYzU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.OfhHCuFtckjkEEyzYlTY6nQQmKx61_oYFkKOwDIBdCk) ![Drone Analog 5](https://private-user-images.githubusercontent.com/176497054/351752702-06492658-4c80-47ba-9fe8-d21d279f7a6a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4NTk0MzYsIm5iZiI6MTcyMTg1OTEzNiwicGF0aCI6Ii8xNzY0OTcwNTQvMzUxNzUyNzAyLTA2NDkyNjU4LTRjODAtNDdiYS05ZmU4LWQyMWQyNzlmN2E2YS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNFQyMjEyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZDg4ZGNlNzdlMjgwZWE4ZmI3MGIxZTMwNzg5YjFiZDM2ZmMzMjUxZjA5ZGQ5ZDZlNDkyNzIzYmI1MWExNzVlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.eNN4ywIQ22YF6kvzv27sVMiLaRM5DrOWdEiz2yKFsyQ) ![PICT0022](https://private-user-images.githubusercontent.com/176497054/351752722-7983bacd-f527-4069-9b15-4544771f274b.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4NTk0MzYsIm5iZiI6MTcyMTg1OTEzNiwicGF0aCI6Ii8xNzY0OTcwNTQvMzUxNzUyNzIyLTc5ODNiYWNkLWY1MjctNDA2OS05YjE1LTQ1NDQ3NzFmMjc0Yi5KUEc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNFQyMjEyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZTZlOGEyMWY1MDczOTgyMDc5NzI4OTg1MmZkODk0MjkxYWM4YTk5MGFhN2I3MjJjYTY3MWI4MTU3NzNlYzUyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.tu69FePKDpqciglGUqH0OzcorJJVSvgAY3bh1g5Fk18)
> 

the drone one looks like a very edge enhanced input (to cheaply replicate that just blur your input + add fake sharpening) with really bad chroma aliasing. i have a clean composite preset that should get you most of the way there, but i was bored so i also decided to make a custom preset that should match what you're after.

[ntsc-rs_cvbsdrone.json](https://github.com/user-attachments/files/16368033/ntsc-rs_cvbsdrone.json)

**Reply by Asyahril395** [reply]
Thanks! :)

**Reply by Degamisu** [reply]
> to cheaply replicate that just blur your input + add fake sharpening

Don't forget fisheye! those drones have a massive fisheye effect which can be easily replicated.

**Reply by Asyahril395** [reply]
OK, I'm trying it (ᴖ◡ᴖ)

**Reply by smfthe1199** [reply]
@nlolnlolnlolnlo did you do the drone effect?

Just making sure as I am upgrading the **NTSC-RT Preset Pack** (I got permission from the creator of this pack).

**Comment by notdigidani** [comment]
you need a MiniDV Sony Handycam. i have a DCR-TRV18E and, from what i've seen, the quality is pretty similar to the examples you've shown. they sell them on Ebay or Depop, or (probably) any online thriftshop you could find. but id recomend asking your parents or grandparents, because they might have one you like

**Reply by Degamisu** [reply]
Any old digital camera would work, but yes. MiniDV is a very good tape format for this. I had a MiniDV camera that recently broke down, but it had amazing quality and was perfect for NTSC-RS.
